### ISSUE TITLE
Create a "getting started" section explaining how to use the typings in a typescript project.

### ISSUE THREAD

**Issue by Diullei** [DefinitelyTyped/definitelytyped.github.io#43]
Create a "getting started" section explaining how to use the typings in a typescript project. Ref: https://github.com/borisyankov/DefinitelyTyped/issues/4631

**Comment by sebastian-lenz** [comment]
I think this is tightly bound to #15 as usage of definition files depends on the IDE you are using. I'm happy to look into this and at least provide some docs for the tools I'm using (IDE free/CLI, WebStorm/PHPStorm, Atom, maybe Sublime) as soon I got time.

**Comment by Diullei** [comment]
@sebastian-lenz 😃

**Comment by jessebarnum** [comment]
Yeah, the one-line 'usage' instructions are extremely frustrating. A more complete example would be very helpful.

**Comment by jamesarosen** [comment]
It would be great to know how to set up `tsconfig.json`. Is
```
{
  "compilerOptions": {
    "typeRoots": ["./types", "./node_modules/@types"]
  }
}
```
the minimum viable usage of DefinitelyTyped? What do you do if a package includes its own type definitions (like `aws-sdk`)?
